ops/terraform/modules/resources/asg/main.tf

```diff
@@ -127,7 +127,7 @@ resource "aws_launch_template" "main" {
     port          = var.lb_config.port
     accountId     = var.launch_config.account_id
     gitBranchName = var.launch_config.git_branch
-    gitCommitId   = var.launch_config.git_branch
+    gitCommitId   = var.launch_config.git_commit
```

Comment:
It appears this was never working as intended, though if it was it would have broken due to the fact that the the commit ID is a unique merge commit that cannot be referenced from anywhere other than in that Jenkins build.  Fixing intended behavior here, but commenting out in the launch template.